src/table/components/TableTotals.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { memo, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import TableRow from '@mui/material/TableRow';
+import TableCell from '@mui/material/TableCell';
+import { useContextSelector, TableContext } from '../context';
+import { getHeaderStyle, getTotalsCellStyle } from '../utils/styling-utils';
+import { totalHeadHandleKeyPress } from '../utils/handle-key-press';
+import { handleClickToFocusHead } from '../utils/handle-accessibility';
+
+function TableTotals({ rootElement, tableData, theme, layout, translator, keyboard }) {
+  const { columns, paginationNeeded, totalsPosition } = tableData;
+  const setFocusedCellCoord = useContextSelector(TableContext, (value) => value.setFocusedCellCoord);
```

Comment:
Ah you are already using it here, good :) just remove the pass you prop in tableWrapper then